test/support.ts

```diff
@@ -245,32 +246,66 @@ export function getAbstractQueryGenerator(sequelize: Sequelize): unknown {
   return new ModdedQueryGenerator({ sequelize, _dialect: sequelize.dialect });
 }
 
+export function getDialectName(dialect: string): Dialect | string {
```

Comment:
Almost all of this code is duplicated with `Config()`, it should be unified so config calls this method. Maybe rename this to `parseDialectEnv` and make it return `{ dialect: Dialect, version: 'latest' | 'oldest' }` so `Config()` can use it

Reply:
I like the way that sounds.  I'll take a look.  Really like what you all are doing with _oldest_ and _latest_.